CodeGeneration/Sources/SyntaxSupport/Child.swift

```diff
@@ -83,6 +84,12 @@ public class Child {
     return lowercaseFirstWord(name: name)
   }
 
+  /// A name of this child that's suitable to be used for variable or enum case names.
+  /// It will first use the deprecated value otherwise the normal one
+  public var deprecatedVarName: String {
+    return lowercaseFirstWord(name: deprecatedName ?? name)
+  }
```

Comment:
Is was mostly to avoid to have many places in the code where we default to `child.varName`. 
I tried to add in the comment too. 

But can revert if that's more clear what happens

Reply:
Yeah, I understand. I think the behavior would be good, I’d just like some other name for it but I don’t have any better ideas.

Reply:
I think it's not that bad using if let and optional `??`.

What do you think